addons/account/demo/account_demo.py

```diff
@@ -61,9 +61,13 @@ def _post_load_demo_data(self, company=False):
     def _get_demo_data_products(self, company=False):
         prod_templates = self.env['product.product'].search(self.env['product.product']._check_company_domain(company))
         if self.env.company.account_sale_tax_id:
-            prod_templates.write({'taxes_id': [Command.link(self.env.company.account_sale_tax_id.id)]})
+            prod_templates_sale = prod_templates.filtered(
```

Comment:
This means that you cannot have demo products without taxes anymore

Reply:
Was not that the case already if you defined a default sale tax?

Reply:
OK: Demo is loaded after template so we would end up with both taxes